commit ad40050d71263adb0b58fa294a30dd06adbc9174 (refs/changes/98/298/1)
Author: David Pacheco <dap@joyent.com>
Date:   2016-08-16T13:08:48-07:00 (3 years, 2 months ago)
    
    joyent/node-verror#40 add contribution guidelines that reference cr.joyent.us (cont)

diff --git a/CONTRIBUTING.md b/CONTRIBUTING.md
index 4192e25..750cef8 100644
--- a/CONTRIBUTING.md
+++ b/CONTRIBUTING.md
@@ -9,8 +9,8 @@ See the [Joyent Engineering
 Guidelines](https://github.com/joyent/eng/blob/master/docs/index.md) for general
 best practices expected in this repository.
 
-Contributions should be "make prepush" clean.  To run "make check", you'll need
-these tools:
+Contributions should be "make prepush" clean.  The "prepush" target runs the
+"check" target, which requires these separate tools:
 
 * https://github.com/davepacheco/jsstyle
 * https://github.com/davepacheco/javascriptlint
